apps/frontend/src/pages/root/components/user-wheel/libs/components/edit-mode-switch/edit-mode-switch.tsx

```diff
@@ -0,0 +1,35 @@
+import { Button } from "~/libs/components/components.js";
+
+import { type EditMode } from "../../types/types.js";
+import styles from "./styles.module.css";
+
+type Properties = {
+	currentMode: EditMode;
+	handleModeToggle: () => void;
+};
+
+const EditModeSwitch: React.FC<Properties> = ({
```

Comment:
I think we should create reusable switch component, because on tasks page there should be switch with the same design to toggle between current and past tasks